# SmacHybrid x MPPI: changes against /opt/ros/jazzy/share/nav2_bringup/params/nav2_params.yaml
# written by parity/scripts/generate_matrix_configs.py; 1 key(s) changed
# planner_server.GridBased.plugin: 'nav2_navfn_planner::NavfnPlanner' -> 'nav2_smac_planner::SmacPlannerHybrid'  (plugin name)
--- installed/nav2_params.yaml
+++ SmacHybrid_MPPI/params.yaml
@@ -375,7 +375,7 @@
   ros__parameters:
     GridBased:
       allow_unknown: true
-      plugin: nav2_navfn_planner::NavfnPlanner
+      plugin: nav2_smac_planner::SmacPlannerHybrid
       tolerance: 0.5
       use_astar: false
     costmap_update_timeout: 1.0
